.buildkite/code-freeze.yml

```diff
@@ -1,15 +1,15 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 ---
 
-agents:
-  queue: "android"
 
 steps:
   - label: "Code Freeze"
     plugins: [$CI_TOOLKIT]
     command: |
-      .buildkite/commands/configure-git-for-release-management.sh
+      source .buildkite/commands/configure-git-for-release-management.sh
```

Comment:
I agree with @mokagio  that we might as well remove `configure-git-for-release-management.sh` at that point and call `source use-bot-for-git` directly from there.

Reply:
Updated on db4b88f08ed274269a27bb76bcb69b40942a31fd (plus added logging on 59a75985689463d990107d1358f955ee5426f83e)